src/reducer.js

```diff
@@ -0,0 +1,37 @@
+import { combineReducers } from 'redux'
+
+const elementsData = (state = {}, action) => {
```

Comment:
I typically see reducers setup like this: https://redux.js.org/basics/example#reducers

I think it's better that way because:
1. For a given action, there should only be one return - a `switch` perfectly models this
2. A `switch` can also catch a default state and return `state`
3. It seems to have less repetition than what you've implemented below (specifically the `else return state` stuff)

Was there a reason for your current implementation?

Reply:
I was warned once that large Switch statements can be hard to debug (i.e. if you miss a return statement or break statement), so I have been avoiding Switch statements in general.... but I do see how that would avoid repetition, and Redux's official docs are using them!